arm-servicebus/2015-08-01/swagger/eventhub.json

```diff
@@ -0,0 +1,1862 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "EventHubManagementClient",
+    "description": "Azure EventHub client",
+    "version": "2014-09-01"
```

Comment:
please make sure that errors pointed in the code review of the previous PR are not repeated in the next PR. It is a time consuming process to review the spec line by line. It is even more frustrating to see the same mistakes being repeated again.